app/uk/gov/hmrc/thirdpartyapplication/domain/models/ApplicationUpdate.scala

```diff
@@ -45,6 +48,7 @@ trait GatekeeperSpecificApplicationUpdate extends ApplicationUpdate {
 }
 case class ChangeProductionApplicationName(instigator: UserId, timestamp: LocalDateTime, gatekeeperUser: String, newName: String) extends GatekeeperSpecificApplicationUpdate
 case class DeclineApplicationApprovalRequest(gatekeeperUser: String, reasons: String, timestamp: LocalDateTime) extends GatekeeperSpecificApplicationUpdate
+case class DeleteApplicationByGatekeeper(actor: Actor, requestedByEmailAddress: String, reasons: String, timestamp: LocalDateTime) extends ApplicationUpdate
```

Comment:
Likewise GatekeeperTiresToDeleteApplication (or wants to or actions or something)